### PR TITLE
More AL2023 preparation changes

### PR DESCRIPTION
**Pull request by pb8o**
## Changes

More changes to prepare AL2023 support

## Reason

Adapt performance tests and gather preliminary perf baselines 

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license. For more information on following
Developer Certificate of Origin and signing off your commits, please check
[`CONTRIBUTING.md`][3].

## PR Checklist

~- [ ] If a specific issue led to this PR, this PR closes the issue.~
- [x] The description of changes is clear and encompassing.
- [x] Any required documentation changes (code and docs) are included in this PR.
~- [ ] API changes follow the [Runbook for Firecracker API changes][2].~
~- [ ] User-facing changes are mentioned in `CHANGELOG.md`.~
- [x] All added/changed functionality is tested.
- [x] New `TODO`s link to an issue.
- [x] Commits meet [contribution quality standards](https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md#contribution-quality-standards).

---

- [ ] This functionality cannot be added in [`rust-vmm`][1].

[1]: https://github.com/rust-vmm
[2]: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md
[3]: https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md

### CODE REVIEW

File: tests/integration_tests/functional/test_cpu_features.py [hunk 1]
```diff
@@ -321,10 +321,10 @@ def test_cpu_rdmsr(
 
     # pylint: disable=C0121
     microvm_val_df = microvm_val_df[
-        microvm_val_df["MSR_ADDR"].isin(MSR_EXCEPTION_LIST) == False
+        ~microvm_val_df["MSR_ADDR"].isin(MSR_EXCEPTION_LIST)
```
Comment (roypat) [inline tests/integration_tests/functional/test_cpu_features.py:324]: nit: commit messages says "use `!`", but you use `~`

**Review by roypat — APPROVED** [review]

File: tests/integration_tests/functional/test_cpu_features.py (hunk 1 above)
Comment (pb8o) [reply tests/integration_tests/functional/test_cpu_features.py:324]: Doh! If I make a new PR I will fix it.

**Review by andreitraistaru — APPROVED** [review]

**Comment by andreitraistaru** [comment]
Have you checked the pipelines' config we get after applying these changes (at a visual review they look good to me but just wanted to double check; anyway, I'm gonna test it on my side too)? Anyway, we need to do some adjustments for the pipelines' config after these changes getting merged.

**Comment by andreitraistaru** [comment]
Ok, looked closer to the BK scripts and looks good. We can go on with the PR from my side.

**Review by sudanl0 — APPROVED** [review]

**Pull request merged**